id-reservations/item-ids.json5

```diff
@@ -15,4 +15,5 @@
   [ -6000,  -6999, "AnnoyingPerson"],
   [ -7000,  -7999, "Cyph"],
   [-69000, -69999, "Penkly"],
+  [-8000, -8999, "Old-FW"],
```

Comment:
Can you move this one line above, so the numbers are in order (-7000, -8000, -69000) and can you add a bit space so the commas allign and then it looks neat. Thank you!

Reply:
Think I've done it correctly now. Let me know if I need to change anything else. Many thanks.